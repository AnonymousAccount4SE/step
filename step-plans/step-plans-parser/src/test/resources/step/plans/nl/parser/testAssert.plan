Function name="test1"
    Script output.add("Name",input.getString('Name'))
EndFunction

test1 Name="Franz"
Assert Name="Franz"