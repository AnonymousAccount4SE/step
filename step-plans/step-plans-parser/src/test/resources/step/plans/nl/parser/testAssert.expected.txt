TestCase:PASSED:
 Session:PASSED:
  CallKeyword:PASSED:
   Sequence:PASSED:
    Script:PASSED:
   Assert:PASSED:
